pom.xml

```diff
@@ -315,26 +349,23 @@
         <artifactId>spotless-maven-plugin</artifactId>
         <version>${plugin.version.spotless}</version>
         <configuration>
-          <java>
-            <googleJavaFormat>
-              <version>${plugin.version.googlejavaformat}</version>
-              <style>GOOGLE</style>
-            </googleJavaFormat>
-          </java>
-          <pom>
-            <sortPom>
-              <expandEmptyElements>false</expandEmptyElements>
-              <spaceBeforeCloseEmptyElement>true</spaceBeforeCloseEmptyElement>
-              <sortProperties>true</sortProperties>
-            </sortPom>
-          </pom>
+          <markdown>
+            <includes>
+              <include>**/*.md</include>
+            </includes>
+            <excludes>
+              <exclude>**/target/**/*.md</exclude>
+              <exclude>clients/go/vendor/**/*.md</exclude>
+            </excludes>
+            <flexmark />
+          </markdown>
```

Comment:
I admit I don't fully understand it myself. I copied what I found in Zeebe. There you have the Java code formatting in parent pom in pluginManagement section and the markdown formatting in root POM plugin section.

The reason for this is that everything in pluginManagement is meant to be inherited, but not apply to the current project.
And everything in plugins section is meant for the current project but not meant to be inherited.

So the markdown is in the root, because the root sees all folders. One issue of the markdown formatting seems to be that it is not aware of subprojects. So by running it in root, we don't need to run it in the sub projects.

At the same time the root doesn't have java sources, so it doesn't need these formatting settings.

What adds to the confusion is that our root project is also the parent project, and that the behavior doesn't fully align with my expectations looking at the docs. For example, SortPOM is needed also in the root and it is also executed. And the documentation for pluginManagement https://maven.apache.org/pom.html#plugin-management states that the settings defined in pluginManagement only apply to children, if they in turn reference the plugin in their plugin section. But it seems to work without that as well.

Maybe @korthout can explain the mystery. 

As for my humble understanding. I am sure it works the way it is configured now and I also believe it is closer to how it is intended by the masters of Maven :-)